modules/wsl-conf.nix

```diff
@@ -0,0 +1,90 @@
+{ lib, pkgs, config, ... }:
+with builtins; with lib; {
+
```

Comment:
```suggestion

with lib; {
```
almost all builtins are inherited into lib.

Reply:
Didn't know that works - thanks